### DISCUSSION TITLE
Is it possible to leave the detection to my Reolink cameras?

### DISCUSSION THREAD

**Discussion by Trailverdeeilmare** [blakeblackshear/frigate#6638] [Ask A Question]
Hello everyone, I would like to know if I can use Frigate without enabling the detection and record and take snapshots using the Reolink detection.

Could be possible?

Thanks

**Comment by NickM-27** [comment]
What are you wanting frigate to do exactly? In general, no, there's no compatibility with third party detection systems

**Reply by Trailverdeeilmare** [reply]
I would like to use the recording/snapshot functionality of Frigate but leave the detection to my Reolink cameras. I'm a newbie but I was thinking to make an automation on HA to to this.

**Reply by NickM-27** [reply]
This will be possible in 0.13 but not now
